metadata-ingestion/tests/unit/sdk_v2/test_lineage_client.py

```diff
@@ -146,12 +143,14 @@ def test_get_strict_column_lineage(client: DataHubClient) -> None:
     ]
 
     # Run test cases
-    for i, test_case in enumerate(test_cases):
+    for test_case in test_cases:
         result = client.lineage._get_strict_column_lineage(
             cast(Set[str], test_case["upstream_fields"]),
             cast(Set[str], test_case["downstream_fields"]),
         )
-        assert result == test_case["expected"], f"Test case {i + 1} failed"
+        assert result == test_case["expected"], (
```

Comment:
not strictly required - but check out pytest parameterize https://docs.pytest.org/en/stable/how-to/parametrize.html#pytest-mark-parametrize